env/scratch/database-tools/terragrunt.hcl

```
@@ -44,8 +44,9 @@ inputs = {
   blazer_image_tag                = "latest"
   database-tools-securitygroup    = dependency.eks.outputs.database-tools-securitygroup
   database-tools-db-securitygroup = dependency.eks.outputs.database-tools-db-securitygroup
+  notify_url                      = "https://staging.notification.cdssandbox.xyz/"
```

Can we modify this to scratch please?

```suggestion
  notify_url                      = "https://scratch.notification.cdssandbox.xyz/"
```